eclipse-collections/src/main/java/org/eclipse/collections/impl/utility/MapIterate.java

```diff
@@ -6,6 +6,18 @@
  * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
  * and the Eclipse Distribution License is available at
  * http://www.eclipse.org/org/documents/edl-v10.php.
+ * 
+ * Utility class for iterating and performing operations on java.util.Map objects.
+ 
```

Comment:
This block of documentation should go above the class type definition. Check the source code of java.lang.String for an example.

Reply:
Sure I will update it ASAP.
Thanks